tsc/process/release.md

```diff
@@ -0,0 +1,20 @@
+**Release Process for OpenVDB**
+
+The following assumes that the current OpenVDB library version number is 6.0.0 and the new version number is 6.1.0. Adjust for the actual version numbers as appropriate.
+
+- [ ] Open a Jira "Release OpenVDB 6.1.0" ticket with "OpenVDB_6.1.0" as the Fix Version.
+- [ ] Update `openvdb/CHANGES` and `openvdb/doc/changes.txt` with release notes.
```

Comment:
Can you clarify - assuming that the release notes have been maintained throughout the commit history since the last release, the only task needed here would be to adjust the release numbers in these changes files (ie "In Development" -> "6.1.0")?

Reply:
If the release notes have been maintained throughout, there should be nothing to do here other than to replace "In Development" with the target release date.  But if we're going to consider alternatives like reno or temporary change logs, then yes, this step should specify precisely what needs to be done to bring the release notes up to date.